pay-api/migrations/versions/b6e28faea978_rename_tables_to_plurals.py

```diff
@@ -0,0 +1,187 @@
+"""rename tables to plurals 
+
+Revision ID: b6e28faea978
+Revises: 90afcc0d6d9f
+Create Date: 2021-01-25 21:50:58.043768
+
+"""
+from alembic import op
+from sqlalchemy import create_engine
+from sqlalchemy.engine import reflection
+from sqlalchemy import MetaData
+import re
+
+# revision identifiers, used by Alembic.
+from sqlalchemy.engine.reflection import Inspector
+
+VERSION = '_version'
+
+# revision identifiers, used by Alembic.
+revision = 'b6e28faea978'
+down_revision = '90afcc0d6d9f'
+branch_labels = None
+depends_on = None
+
+###
+# get_pk_constraint
+
+table_mapping = {'bcol_payment_account ': 'bcol_payment_accounts',
```

Comment:
bcol_payment_account, credit_payment_account and internal_payment_account are not in use, we kept it there as a backup.